guide/english/html/comments-in-html/index.md

```diff
@@ -38,7 +38,7 @@ Conditional Comments are only recognised by Internet Explorer Version 5 through
 <!DOCTYPE html>
 <html>
 	<body>
-		<!--[if IE 9]>
+		<!--[if IE 9]-->
```

Comment:
While your addition of `<![endif]-->` is correct, you should not have added the `--`  to line 41.  Please correct.

Thank you.